### DISCUSSION TITLE
Question regarding display of BT-120-Lot on TED

### DISCUSSION THREAD

**Discussion by lesomue** [OP-TED/eForms-SDK#892] [Q&A]
Hello, 

we recognise some inconsistencies in the display of notices on TED regarding BT-120 „The buyer reserves the right to award the contract on the basis of the initial tenders without any further negotiations“. 

In one of our notices,  above-mentioned indication is missing in the TED notices, which should be displayed under 5.1.9 (please see https://ted.europa.eu/en/notice/-/detail/688044-2023). When checking the respective XML file downloaded from TED direcly, we cannot find any error or mistake and the respective BT seems to be correctly shown (please see [XML file](https://ted.europa.eu/en/notice/688044-2023/xml)): 

![image](https://github.com/OP-TED/eForms-SDK/assets/117835335/e318db53-8ff8-4940-ab05-3d01239cc61d)

In another notice, the information is given just as expected and we can not identify any difference to the other notice with regard to BT-120-Lot (please see https://ted.europa.eu/en/notice/-/detail/172247-2024 and the [XML file](https://ted.europa.eu/en/notice/172184-2024/xml)). 

Could you please let us know if there is any mistake in the XML file we can correct or why the inconsistencies come up? 

We have already placed this question via the TED Helpdesk but hope to find a quicker answer here. So many thanks in advance. 

Best regards
Lea Müller
cosinex GmbH

**Comment by pdonohoe** [comment]
Hi @lesomue, this difference is not related to an issue with the SDK, so I am moving this to a Conversation. I will comment further there.

**Comment by pdonohoe** [comment]
Hi @lesomue, I agree, the two XML files have the same information and structure for BT-120-Lot, so both should be displayed. Although both XML files have errors, I don't think any of them are related to this display issue. 
This is not the correct place to discuss this issue. I will request the Helpdesk to assign your ticket to me.

**Reply by lesomue** [reply]
Many thanks!
